kernel/indtypes.ml

```diff
@@ -908,10 +505,10 @@ let build_inductive env prv iu env_ar paramsctxt kn isrecord isfinite inds nmr r
     let rtbl = Array.init (List.length cnames) transf in
       (* Build the inductive packet *)
       { mind_typename = id;
-	mind_arity = arkind;
-	mind_arity_ctxt = Vars.subst_univs_level_context substunivs ar_sign;
-	mind_nrealargs = Context.Rel.nhyps ar_sign - nparamargs;
-	mind_nrealdecls = Context.Rel.length ar_sign - nparamsctxt;
+        mind_arity = arity;
+        mind_arity_ctxt = indices @ paramsctxt;
+        mind_nrealargs = Context.Rel.nhyps indices;
```

Comment:
This and the next used to be two different contexts, and now they are the same? Just trying to understand the change.